### PR TITLE
Rails 5 support

### PR DESCRIPTION
**Pull request by marcroberts**

### CODE REVIEW

**Comment by marcroberts** [comment]
I've removed some of the older rubies as you indicated we could drop the legacy versions in #43

**Comment by ssaunier** [comment]
Hi guys, any progress on this? Do you want some help? Thanks!

**Comment by mibamur** [comment]
+1

**Comment by mibamur** [comment]
is it the same https://github.com/evrone/quiet_assets/pull/43 ?

**Comment by berfarah** [comment]
Is this still active?

**Comment by route** [comment]
QA is in rails 5 from now on

**Comment by nbulaj** [comment]
@route can you please give a link? Can't found it the [Release Notes](http://guides.rubyonrails.org/5_0_release_notes.html)

**Comment by route** [comment]
@nbulaj it's somewhere deep inside sprockets-rails. It should work by default though

**Comment by route** [comment]
It's all started here https://github.com/rails/sprockets-rails/pull/338 follow it

**Comment by kmcphillips** [comment]
Implemented here: https://github.com/rails/sprockets-rails/pull/355

Deprecation message here: #51

**Comment by nbulaj** [comment]
tl;dr

```ruby
# config/environments/development.rb

# Suppress logger output for asset requests.
config.assets.quiet = true
```
